frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboard.tsx

```diff
@@ -15,6 +16,15 @@ interface MonitoringDashboardProps {
 
 const MonitoringDashboard: React.FC<MonitoringDashboardProps> = ({ match }) => {
   const namespace = match.params.ns;
+  const params = getURLSearchParams();
+  const { workloadName, workloadType } = params;
+  let queries: MonitoringQuery[];
+
+  if (workloadName && workloadType) {
+    queries = workloadMetricQueries;
+  } else {
+    queries = monitoringDashboardQueries;
+  }
```

Comment:
this can written as `const queries: MonitoringQuery[] = workloadName && workloadtype ? workloadMetricQueries : monitoringDashboardQueries`